subsys/bluetooth/audio/bap_stream.c

```diff
@@ -406,7 +406,11 @@ void bt_bap_stream_detach(struct bt_bap_stream *stream)
 	stream->ep = NULL;
 
 	if (!is_broadcast) {
-		bt_bap_stream_disconnect(stream);
+		const int err = bt_bap_stream_disconnect(stream);
+
+		if (err != 0) {
+			LOG_DBG("Failed to disconnect stream %p: %d", stream, err);
```

Comment:
should it have been LOG_ERR?

Reply:
I was considering that, but it is also called during stream-reset, in which case we may not have an ACL, and this would always fail in that case, and we wouldn't want to log it as an error in that case. 

The alternative to this logging would be to simply ignore the return value.

Reply:
ok.. true